tests/integration/integrationtests.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import sys
+import unittest
+from random import randint
+
+import utils
+
+
+class IntegrationTest(unittest.TestCase):
+    # helper funtion: test response time
+    def assert_response_time(self, elapsed_seconds, max_elapsed_seconds):
+        logging.debug(f"Request took {elapsed_seconds} second(s)")
+        self.assertLess(elapsed_seconds, max_elapsed_seconds)
+
+    def test_bad_request(self):
+        res = utils.get_by_params(None)
+        self.assert_response_time(res.elapsed.total_seconds(), 1)
```

Comment:
every call you have to `assert_reponse_time` contains `res.elapsed.total_seconds()`. Instead, how about passing the whole requests object so that `assert_response_time` can take care of getting the elapsed seconds?

Reply:
Yeah, we could, but I'm just curious is there any benefit to do so?

Reply:
Well, never mind. It did make the code looks better :)